### PR TITLE
Add e2e test update

### PR DESCRIPTION
**Pull request by neallred**
#### Summary
Fills out e2e test for onboarding tutorial tip. These are changes that were mentioned, but accidentally not included, in https://github.com/mattermost/mattermost-webapp/pull/8544 . QA not needed for this PR: it adds implementation of an e2e test, but nothing else.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37472

#### Related Pull Requests
NONE

#### Screenshots

https://user-images.githubusercontent.com/13738432/129080556-a9efc91c-9a0e-475f-af3e-ac27b47151a7.mp4

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by pablovelezvidal — APPROVED** [review]
LGTM! :shipit:

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 1]
```diff
@@ -9,66 +9,48 @@
 
 // Stage: @prod
 // Group: @onboarding @smoke
+const sel = {
```
Comment (pablovelezvidal) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:12]: nit: could we add a more descriptive name than sel? selectors would be ok, too.

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 2]
```diff
@@ -9,66 +9,48 @@
...
-                cy.apiLogin(testUser);
-                cy.visit(`/${testTeam.name}/channels/town-square`);
+                cy.apiLogin(user);
+                cy.visit(`/${team.name}/channels/town-square`);
             });
         });
     });
 
     it('Takes the user through the steps of using the app', () => {
         // tip 1: post message
```
Comment (AGMETEOR) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:32]: Top of the file has a guide on how to make the comments

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 3]
```diff
@@ -9,66 +9,48 @@
...
-        cy.get('#create_post #tipButton').click();
-
-        cy.get('.tip-overlay').should('be.visible').
-            and('contain', 'Type your first message and select Enter to send it.');
-
-        cy.get('#tipNextButton').click();
+        cy.get(`#create_post ${selectors.tip}`).click();
```
Comment (AGMETEOR) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:33]: We can add comments on every code block to tell what the intended action is like the existing e2e files

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 4]
```diff
@@ -9,66 +9,48 @@
...
+        cy.get(selectors.tipText).should('be.visible').
+            and('contain', 'Send a message');
+        cy.get(selectors.tipNext).click();
 
         // tip 2: channels
```
Comment (AGMETEOR) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:38]: Top of the file has a guide on how to make the comments

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 5]
```diff
@@ -9,66 +9,48 @@
...
-        // cy.get('.tip-overlay').should('be.visible').
-        //     and('contain', 'Type your first message and select Enter to send it.');
-        //
-        // cy.get('#tipNextButton').click();
+        cy.get(`#sidebarItem_town-square ${selectors.tip}`).click();
+        cy.get(selectors.tipText).should('be.visible').
+            and('contain', 'Organize conversations in channels');
+        cy.get(selectors.tipNext).click();
 
         // tip 3: add channels
```
Comment (AGMETEOR) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:44]: Top of the file has a guide on how to make the comments

File: e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js [hunk 6]
```diff
@@ -9,66 +9,48 @@
...
-        // tip 4: admin
+        cy.get(`#lhsNavigator ${selectors.tip}`).click();
+        cy.get(selectors.tipText).should('be.visible').
+            and('contain', 'Create and join channels');
+        cy.get(selectors.tipNext).click();
+
+        // tip 4: invite people/admin
```
Comment (AGMETEOR) [inline e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js:50]: Top of the file has a guide on how to make the comments

**Review by AGMETEOR — CHANGES_REQUESTED** [review]

**Comment by marianunez** [comment]
Adding @stevemudie here for tests visibility and help with Zephyr test numbers and here: https://github.com/mattermost/mattermost-webapp/pull/8544#discussion_r689443633

**Review by AGMETEOR — APPROVED** [review]
Nice @neallred . LGTM

**Comment by amyblais** [comment]
@neallred Is this a requirement for v6.0? If yes, can you help cherry-pick to the ``cloud`` branch? https://community.mattermost.com/core/pl/cmqefh9ue7dnxgfm1cirbcj9se

**Pull request merged**